go/libkb/saltpack_enc.go

```diff
@@ -54,7 +54,7 @@ func SaltpackEncrypt(g *GlobalContext, arg *SaltpackEncryptArg) error {
 	var err error
 	if !arg.EncryptionOnlyMode {
 		if arg.SaltpackVersion.Major == 1 {
-			return errors.New("specifying the saltpack version 1 requires --current-devices-only")
+			return errors.New("specifying the saltpack version 1 requires --auth-type=repudiable")
```

Comment:
I'm never a huge fan of moving knowledge of flags out of `client/` and into `libkb`, but i realize you inherited this wart.